sceptre/stack.py

```diff
@@ -22,103 +23,88 @@
 from sceptre.template import Template
 
 
+class LaunchAction(Enum):
+    include = 1  # This is the default value for all stacks. It means launch runs create/update.
+    exclude = 2  # This means launch won't create and will delete it if it exists.
+
+
 class Stack(object):
     """
     Stack stores information about a particular CloudFormation Stack.
 
     :param name: The name of the Stack.
-    :type project: str
```

Comment:
These type docstrings aren't necessary any more since we're already used type annotations on the parameters.